scripts/make_compmusic_jingju_acappella.py

```diff
@@ -0,0 +1,147 @@
+import argparse
+import hashlib
+import json
+import os
+
+
+JINJGU_ACAPPELLA_INDEX_PATH = (
+    "../mirdata/datasets/indexes/compmusic_jingju_acappella_index.json"
+)
+
+
+def md5(file_path):
+    """Get md5 hash of a file.
+
+    Parameters
+    ----------
+    file_path: str
+        File path.
+
+    Returns
+    -------
+    md5_hash: str
+        md5 hash of data in file_path
+    """
+    hash_md5 = hashlib.md5()
+    with open(file_path, "rb") as fhandle:
+        for chunk in iter(lambda: fhandle.read(4096), b""):
+            hash_md5.update(chunk)
+    return hash_md5.hexdigest()
```

Comment:
We should use the validation version of `md5`, I know it's all over the place in the scripts, but maybe from now on?